packages/sample/modals/src/app.module.ts

```diff
@@ -0,0 +1,33 @@
+import { DiscordModule } from '@discord-nestjs/core';
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { default as discordModals } from 'discord-modals';
+import { Client, Intents } from 'discord.js';
+
+import { BotModule } from './bot/bot.module';
+
+@Module({
+  imports: [
+    ConfigModule.forRoot(),
+    DiscordModule.forRootAsync({
+      imports: [ConfigModule],
+      useFactory: (configService: ConfigService) => ({
+        token: configService.get('TOKEN'),
+        discordClientOptions: {
+          // https://stackoverflow.com/questions/64006888/can-anyone-explain-the-issue
+          intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES, 32767],
```

Comment:
I'm curious. The id `32767`, is that something that hasn't been released on `discord.js`? Would it make sense to make internal `Intents` and apply this id?

Reply:
The id `32767` means `ALL_INTENTS`, discord.js allows you to use number instead of direct FLAGS
If you want to know more 
- https://stackoverflow.com/questions/68820429/discord-intents32767-vs-intents-flags-guilds
- https://discordjs.guide/popular-topics/intents.html#more-on-bitfields

Reply:
> I'm curious. The id `32767`, is that something that hasn't been released on `discord.js`? Would it make sense to make internal `Intents` and apply this id?

When I did I just copied the intention from [here](https://www.npmjs.com/package/discord-modals#:~:text=const%20client%20%3D%20new%20Client(%7B%20intents%3A%2032767%20%7D))

Reply:
In general it was redundant